src/pybind/mgr/feedback/feedbackServices.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+
+import json
+import requests
+
+from .feedbackModel import Feedback
```

Comment:
In python it's not common to use CamelCase for files. Something like `mgr/feedback/model.py`, `mgr/feedback/services.py` is perfectly fine.